beacon-chain/powchain/log_processing_test.go

```diff
@@ -410,140 +411,8 @@ func TestProcessETH2GenesisLog_8UniquePubkeys(t *testing.T) {
 	hook.Reset()
 }
 
-func TestUnpackETH2GenesisLogData_OK(t *testing.T) {
```

Comment:
i use this test in my pr, just name change. i have to check trigger has occurred

Reply:
nm, remove it. it doesn't belong here anymore